tests/dut_console/test_escape_character.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import pexpect
+import logging
+import time
+import re
+
+from tests.common.helpers.assertions import pytest_assert
+
+TOTAL_PACKETS = 100
+logger = logging.getLogger(__name__)
+
+def test_console_escape(duthost_console):
+    """
+    Test reverse SSH are working as expect.
+    Verify serial session is available after connect DUT via reverse SSH
+    """
+    try:
+        child = pexpect.spawn("ping 127.0.0.1 -c {} -i 1".format(TOTAL_PACKETS))
+        time.sleep(5)
+        child.sendcontrol('C')
+        child.expect("\^C")
+        match = re.search(r'(\d) packets transmitted', child.read())
+        pytest_assert(int(match.group(1)) < TOTAL_PACKETS, "Escape Character does not work.")
+    except Exception as e:
```

Comment:
Can we try to make the Exception more specific, such as something like `ConnectionError`? It is because some assert in pytest is also treated as Exception, which is not supposed to be caught as connection error.

Reply:
I think this try...catch is not necessary here. If the connection is failed, it will fail in the fixture `duthost_console`. So I remove this.